### DISCUSSION TITLE
Question: Should Point3D and List to be observable?

### DISCUSSION THREAD

**Discussion by phamvanthanh** [FXyz/FXyz#160] [Ideas]
Hi,
This is for the purpose of empowering Bindings and Listeners to manipulate shapes with mouse events.

Is thit not a good idea? If yes. why?

**Comment by Birdasaur** [comment]
Hi Pham. Are you thinking about having properties that are connected to the Point3D X, Y, and Z coordinates?

**Comment by phamvanthanh** [comment]
@Birdasaur 
Thanks for your reply. 
Yes, exactly.
For example, the PolyLine3D, how to re-render on changes of points and/or on any point?

**Comment by Birdasaur** [comment]
@phamvanthanh Are you using a modular Java build... ie... JDK 11 or are you using JDK 8?

**Comment by phamvanthanh** [comment]
I am using JDK 11.

**Comment by Birdasaur** [comment]
Have you tried the new 0.5.0 release we pushed out this week?
I could add some SimpleDoubleProperty objects that automatically update whenever the Point3D coordinates change, and then push it as a minor update to 0.5
